ros2_dependencies.bash

```diff
@@ -3,11 +3,12 @@
 set -euf -o pipefail
 
 apt update
+apt -y upgrade
```

Comment:
I'd rather leave this one out so as to keep the image smaller.

Reply:
Yeah, sorry that was supposed to be a separate commit but I missed it. Will remove it and repush.